eng/scripts/tests/Invoke-GenerateAndBuildV2.tests.ps1

```diff
@@ -0,0 +1,52 @@
+#Requires -Version 7.0
```

Comment:
Should these files be under eng/automation/test instead? It would be nice to keep them all in the same directory.

Reply:
Thanks, but I think it is better to put the test the same place with the script itself, then we can easy find out the test script.